docs/source/integrations/cvat.rst

```diff
@@ -1462,6 +1465,66 @@ For example, let's upload some blurred images to CVAT for annotation:
    :alt: cvat-alt-media
    :align: center
 
+
+Linking the occluded property widget
+------------------------------------
+
+The CVAT UI provides a series of widgets on individual labels that control
+properties like occluded, hidden, locked, pinned, etc. The occluded property is
+special in that it is not merely a GUI feature, it is also exposed in the
+backing annotation API. 
+
+This allows you to link a boolean attribute of your labels in FiftyOne with
+this occlusion widget. Annotators can then set value of this attribute purely
+through clicking the widget.
+
+Things to note about the occlusion widget:
+
+* It only supports boolean fields
+* It can only be linked to one attribute per class
+* It is set in the `label_schema` by specifying the `type` of the attribute as
+  `occluded_widget`
+* It can be used to edit an existing boolean attribute or to create a new one
+
+
+.. code:: python
+    :linenos:
+
+    import fiftyone as fo
+    import fiftyone.zoo as foz
+
+    dataset = foz.load_zoo_dataset("quickstart").clone()
+    view = dataset.take(1)
+
+    anno_key = "cvat_occluded_widget"
+
+    # Create a new attribute on existing labels that is
+    # annotated with the occlusion widget 
+    label_schema = {
+        "ground_truth": {
+            "attributes": {
+                "occluded": {
+                    "type": "occluded_widget",
```

Comment:
How about `type="occlusion"` or `type="occluded"` for this? Shorter

Reply:
Gonna go with `"occluded"`